CHANGELOG.md

```diff
@@ -2,11 +2,13 @@
 
 ## (Unreleased)
 
-- Add New Property Properties to TaskOrchestrationContext by @nytian in [#415](https://github.com/microsoft/durabletask-dotnet/pull/415)
-- Add automatic retry on gateway timeout in `GrpcDurableTaskClient.WaitForInstanceCompletionAsync` in [#412](https://github.com/microsoft/durabletask-dotnet/pull/412))
+- Change user-agent header to x-user-agent ([#422](https://github.com/microsoft/durabletask-dotnet/pull/422))

```

Comment:
Rather than writing a new entry, replace or update the one below on line 9 (PR 417).